CMakeLists.txt

```diff
@@ -231,43 +231,33 @@ install(TARGETS ${OpenBLAS_LIBNAME}
 
 # Install include files
 	set (GENCONFIG_BIN ${CMAKE_BINARY_DIR}/gen_config_h${CMAKE_EXECUTABLE_SUFFIX})
-	ADD_CUSTOM_COMMAND(
-	OUTPUT ${CMAKE_BINARY_DIR}/openblas_config.h
-	DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/config.h
-	COMMAND ${GENCONFIG_BIN} ${CMAKE_CURRENT_SOURCE_DIR}/config.h ${CMAKE_CURRENT_SOURCE_DIR}/openblas_config_template.h > ${CMAKE_BINARY_DIR}/openblas_config.h
-	)
 
-	ADD_CUSTOM_TARGET(genconfig
-	ALL
-	DEPENDS openblas_config.h
-	)
-	add_dependencies(genconfig ${OpenBLAS_LIBNAME})
+	execute_process(COMMAND ${GENCONFIG_BIN}
+	                        ${CMAKE_CURRENT_SOURCE_DIR}/config.h
```

Comment:
This has caused issues with out-of-source builds. config.h is located in the ${PROJECT_BINARY_DIR}